docs/use-case-analysis-and-documentation.md

```diff
@@ -0,0 +1,99 @@
+# Use Case Analysis and Documentation
+
+## Unconditional Social Cash Transfer
```

Comment:
We already have section about [USCT](https://github.com/GovStackWorkingGroup/sandbox/blob/main/docs/demo.md#unconditional-social-cash-transfer)
I can try to make it more coherent, but I'm currently busy with Mosip on borading.